http-clients/netty-nio-client/src/main/java/io/netty/handler/codec/http2/ForkedHttp2StreamChannelBootstrap.java

```diff
@@ -44,6 +45,9 @@
     private final Channel channel;
     private volatile ChannelHandler handler;
 
+    // Cache the ChannelHandlerContext to speed up open(...) operations.
+    private volatile ChannelHandlerContext multiplexCtx;
+
     public ForkedHttp2StreamChannelBootstrap(Channel channel) {
```

Comment:
Do you know what the changes we made in the class are compared to the one shipped with Netty? Would be cool if we could get rid of this one too!

Reply:
Verified the diff between our forked file and 4.1.36 file. Both are same except the naming changes. Safe to delete this class too. will update pr.